api/python/api/app.py

```
@@ -624,3 +624,158 @@
+            activity_score += ascent_km * 0.05
+
+    # Race bonus - tiny
+    if activity.race:
+        activity_score *= 1.05
+
+    # Power/intensity contribution - minimal
+    if activity.training_stress_score:
+        activity_score += activity.training_stress_score * 0.005
+    elif activity.avg_power and activity.sport == "cycling":
+        # Rough TSS estimation for cycling without TSS
+        if activity.avg_power > 200:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)